ComplianceAsCode/content
# Fix rule ubtu 20 010070

#### Description:

- Add UBTU-20-010070
- Modify Ansible remediation to include ubuntu 2004
- Deprecate `ubuntu.xml` to use modified `shared.xml` OVAL definition

#### Rationale:

- Part of Ubuntu 20.04 DISA STIG v1r9 profile upgrade
- https://github.com/ComplianceAsCode/content/pull/10738

#### Review Hints:

Build the product:
```
./build_product ubuntu2004
```
To test these changes with Ansible:
```
ansible-playbook build/ansible/ubuntu2004-playbook-stig.yml --tags "DISA-STIG-UBTU-20-010070"
```
To test changes with bash, run the remediation section: `xccdf_org.ssgproject.content_rule_accounts_password_pam_unix_remember`

Checkout Manual STIG OVAL definitions, and use software like DISA STIG Viewer to view definitions.
```
git checkout dexterle:add-manual-stig-ubtu-20-v1r9
```

This STIG can not be tested with the latest Ubuntu 2004 Benchmark SCAP. Please perform a manual test given the check text. For reference, please review the latest artifacts: https://public.cyber.mil/stigs/downloads/
